data/human/human missions.txt

```diff
@@ -7453,3 +7453,1280 @@ mission "Adelita 5"
+			`	There are torn suitcases, clothes and various personal objects strewn throughout the container. The place has been ransacked and the equipment appears to be missing.`
+			`	Tavish rubs his nose and paces back and forth.`
+			`	"Stole it." Kormak says dryly.`
+			`	"Yes, yes, I know. I suppose we should have accompanied him. Don't say it!" Tavish says with irritation.`
+			`	"Told you so." Kormak says defiantly.`
+			`	"I know! But you have to admit the bird watching seminar we attended on New India instead was quite extraordinary!"`
+			`	"Lots of birds." Kormak admits.`
+			`	Tavish takes a deep breath,"Okay, we know the captain's name and his ship's name. I am certain the spaceport authorities have some information about him."`
+			`	"Confidential." Kormak says.`
+			`	"We'll see about that. Let's go." Tavish storms out of the ravished container.`
+			choice
+				`	(Hurry after him.)`
+					goto hurry
+				`	(Stay.)`
+			`	Kormak's iron grip lands on your shoulder. "Come."`
+			`	He pulls you along in Tavish's wake.`
+
+			label hurry
+			`	Tavish easily finds the administrative sector of the spaceport and immediately accosts the first blank-looking official he sees. "I am sorry, Dr. Tavish. But that information is not available to civilians - regardless of who they may claim to be," she replies.`
+			`	"But come now, dear! I can already tell we're kindred spirits, you and I, that you get..." Before Tavish has a chance to finish, a menacing spaceport guard approaches.`
+			`	"You heard her. Move along now," the guard says.`
+			`	Tavish sighs, holds his hands up and backs away. "Your turn," he mouths to Kormak.`
+			`	Kormak steps up to the kiosk, and eyes the still menacing guard.`
+			`	"Ser," he says to the woman, using the old Deep honorific- then lightning quick, his hand lashes out and snatches the surprised guard by the throat. The guard squirms, gasping for air as Kormak holds him up a few inches from the ground. "Please, tell us everything you know about Hans Fuller and his ship, The Broken Record."`
+			`	The area around you is momentarily frozen, shocked by Kormak's quick and easy violence. "Tell us what we need. And this man will be fine. Call for help, or refuse us, and this will get messy for everyone."`
+			`	The guard is so red in the face it looks like he's about to pass out. "O-okay. Just don't hurt anyone, please!" The official moves to her terminal, strikes her keyboard for a few moments and then looks back up. "It s-says he's been impounded multiple times by for selling contrband from Deadman's Cove. Th-that's it." `
+			`	Kormak lets the now unconscious guard's body slump to the floor. "Thank you, ser." Kormak tips his hat to her and turns to Tavish.`
+			`	Tavish sighs, "Splendid work, though I do think we should run now."`
+			`	He turns to you. "We'll meet you at your ship."`
+			`	The two of them then sprint off before the authorities have time to respond.`
+			choice
+				`	(Head back to your ship.)`
```

Comment:
It feels odd to me to force the player into action like that. I can think of many games I've played where there is no choice, only one path to take, and the game makes me take that path. So, even though I have no agency, I am still in taking the action.

Reply:
Perhaps you could add a "Wander around the spaceport for a while" option? Still leads to the same dialog eventually, but tells a little less forced?
~~definitely not saying this because I like your writing~~